frontend/src/components/Card.tsx

```diff
@@ -73,6 +73,11 @@ const Card = ({
             )}
         </div>
       </div>
+      {projectName && (
+        <a href={projectLink} target="_blank" rel="noopener noreferrer" className="font-bold">
```

Comment:
With longer title I noticed that the meta data block gets separated from the edge of the card. If you look closely on this card the left border for the number of comments and date updated block is not touching the top of the card. It should not depend on the size of the title and always "connect" with the card.
Let me know if you need help with updating that too!
![Screenshot 2024-12-23 at 3 03 18 PM](https://github.com/user-attachments/assets/9391c329-172d-45d5-a99b-b88d74b45a78)